### ISSUE TITLE
Class 'Isolated\Symfony\Component\Finder\Finder' not found using Phar 0.13.0

### ISSUE THREAD

**Issue by polevaultweb** [humbug/php-scoper#368]
| Question    | Answer
| ------------| ---------------
| Box version | 0.13.0 
| PHP version | 7.2.1
| Platform with version | MacOS
| Github repo |  https://github.com/polevaultweb/php-scoper-phar-issue

<details>
 <summary>scoper.inc.php</summary>
 
 ```php
 <?php

declare(strict_types=1);

use Isolated\Symfony\Component\Finder\Finder;

return [
   'finders' => [
       Finder::create()->files()->in('src'),
   ],
];

 ```
</details>

<details>
 <summary>Output</summary>
 
 ```bash
 $ PHAR_URL="https://github.com/humbug/php-scoper/releases/download/0.13.0/php-scoper.phar"
$ curl -O -L $PHAR_URL
 
$ php ./php-scoper.phar add-prefix
 > output
[07-Jan-2020 12:35:54 UTC] PHP Fatal error:  Uncaught Error: Class 'Isolated\Symfony\Component\Finder\Finder' not found in /Applications/MAMP/htdocs/my-site/wp-content/plugins/my-plugin/build/scoper.inc.php:14
Stack trace:
#0 phar:///Applications/MAMP/htdocs/my-site/wp-content/plugins/my-plugin/build/php-scoper.phar/src/Configuration.php(58): include()
#1 phar:///Applications/MAMP/htdocs/my-site/wp-content/plugins/my-plugin/build/php-scoper.phar/src/Console/Command/AddPrefixCommand.php(205): _HumbugBox1b62df0e3880\Humbug\PhpScoper\Configuration::load('/Applications/M...')
#2 phar:///Applications/MAMP/htdocs/my-site/wp-content/plugins/my-plugin/build/php-scoper.phar/src/Console/Command/AddPrefixCommand.php(57): _HumbugBox1b62df0e3880\Humbug\PhpScoper\Console\Command\AddPrefixCommand->retrieveConfig(Object(_HumbugBox1b62df0e3880\Symfony\Component\Console\Input\ArgvInput), Object(_HumbugBox1b62df0e3880\Symfony\Component\Console\Output\ConsoleOutput), Object(_HumbugBox1b62df0e3880\Symfony\Component\Console in /Applications/MAMP/htdocs/my-site/wp-content/plugins/my-plugin/build/scoper.inc.php on line 14
 ```
</details>

**Comment by vits** [comment]
Hi, this is not 0.13.0 related. I have same problem with all previous versions down to 0.12.0.

If there is no scoper.inc.php file everything works fine... just uses random prefix and default config.

**Comment by theofidry** [comment]
That sounds really curious...

**Comment by vits** [comment]
Did some research. Problem occurs only with PHAR version (which is preferred installation method). I removed all global composer packages and tried it on empty project to be absolutely sure nothing else conflicts. No matter what, php-scoper.phar gets `Class 'Isolated\Symfony\Component\Finder\Finder' not found`.

However after some effort I was able to install php-scoper as global package and it works as expected.
